.github/workflows/ci.yml

```diff
@@ -60,9 +60,23 @@ jobs:
           - "json:lint"
           - "php:sniff"
           - "php:codestyle"
-          - "php:copypaste"
         php-version:
           - 7.4
+  copypaste-check:
+    name: "Copy paste check"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: "Install PHP"
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          coverage: none
+      - name: "Download tool"
+        run: "wget https://phar.phpunit.de/phpcpd.phar"
+      - name: "Run command"
+        run: "php phpcpd.phar --fuzzy Classes Configuration Tests"
```

Comment:
Instead of downloading the PHAR on every CI run, we should put the PHAR in `tools/` (via PHIVE) and exclude that directory via `.gitattributes` from releases.

Reply:
Like here: https://github.com/MyIntervals/emogrifier